build-tools/src/main/kotlin/org/jetbrains/kotlin/benchmark/BenchmarkingPlugin.kt

```diff
@@ -140,7 +141,6 @@ open class BenchmarkingPlugin: Plugin<Project> {
                     runTask!!.args(
                         "-w", nativeWarmup,
                         "-r", attempts,
-                        "-o", buildDir.resolve(nativeBenchResults).absolutePath,
```

Comment:
With this patch we don't use this task to run benchmarks. But we still configure args here and then read these configured args in the `runKonan` task. It looks like a redundant step and I think it's better to configure `runKonan` directly.

We can even hide the default run task by assigning it to a `null` task group. Also it's possible to disable execution of this task using the corresponding property:
```
runTask.enabled = false
```